app/models/user.rb

```diff
@@ -9,7 +9,10 @@ class User < ApplicationRecord
   has_many :user_venues, as: :manager
   has_many :venues, through: :user_venues
 
+  has_many :reviews, as: :reviewable
+
   def manager_has_venues_with_wine?(wine)
     !(venues & wine.venues).empty?
```

Comment:
Took out "as reviewable" to see if there was some interaction there, still failing. I'll keep trying to debug